core/modules/ckeditor5/js/ckeditor5-prepare.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * @file
+ * Backwards compatibility of UMD build to integration via DLL builds.
+ */
+(function () {
+  "use strict";
+
+  // Mimic some of the DLL structure by adding some aliases for commonly
+  // extended classes. It won't be possible to mimic the entire DLL structure.
+  // This must happen before any custom plugin js files load.
+  CKEDITOR.core = {};// Does not exist in UMD.
+  CKEDITOR.core.Plugin = CKEDITOR.Plugin;
+  CKEDITOR.core.Command = CKEDITOR.Command;
+
+  CKEDITOR.ui = {};// Does not exist in UMD.
+  CKEDITOR.ui.ButtonView = CKEDITOR.ButtonView;
```

Comment:
I wonder if we should switch to the new locations in our core plugins. We currently also rely on that bc layer - which would still be required for contrib/custom. But in core we could just switch.